docs/src/main/asciidoc/scheduler.adoc

```diff
@@ -97,8 +107,17 @@ public class CounterBean {
 (Quarkus is using 10 worker threads for the scheduler). If it is not available the method invocation should be re-scheduled by default i.e
 it should be invoked as soon as possible. The invocation of the scheduled method does not depend on the status or result of the previous invocation.
 4. The code is pretty straightforward. Every 10 seconds, the counter is incremented.
+5. Define a job with a cron-like expressions. The syntax is currently based on Quartz Cron Trigger. The annotated method is executed at 10:15am every day.
```

Comment:
```suggestion
5. Define a job with a cron-like expression. The syntax is currently based on Quartz Cron Trigger. The annotated method is executed at 10:15am every day.
```
Maybe we could add a link to the Quartz documentation?

Reply:
Done